go.mod

```diff
@@ -32,3 +32,5 @@ require (
 	labix.org/v2/mgo v0.0.0-20140701140051-000000000287 // indirect
 	launchpad.net/gocheck v0.0.0-20140225173054-000000000087 // indirect
 )
+
+replace github.com/zorkian/go-datadog-api => github.com/DataDog/go-datadog-api v0.0.0-20190529090230-16add2864293
```

Comment:
remove this line when PR on go-datadog-api is merged

Reply:
You can use `go get github.com/zorkian/go-datadog-api@e55ea7f && go mod vendor` to do the update and get rid of this.